okon.py

```diff
@@ -0,0 +1,44 @@
+""" Implementation of an Abstract Base Class (ABC) Okon 
+
+Okon ABC implementation provides unified communication between
+the autonomy module and simulation/TCM.
+"""
+from abc import ABC, abstractmethod
```

Comment:
tbh I've never seen usage of ABC module in python. I think we don't need to write such a secure code when only few people are going to use it, but if you implemented it this way, it can stay

Reply:
BTW try to remember about clicking "ready for review" from draft